pkg/rolling/options.go

```diff
@@ -253,3 +244,38 @@ func (o *RestartOptions) GetNodeIds() ([]uint32, error) {
 
 	return ids, nil
 }
+
+func parseVersionFlag(versionUnparsedFlag string) (options.VersionSpec, error) {
+	majorMinorPatchPattern := `^(>|<|!=|~=)(\d+|\*)\.(\d+|\*)\.(\d+|\*)$`
+	re1 := regexp.MustCompile(majorMinorPatchPattern)
```

Comment:
It's highly recommended to move MustCompile regexps out of functions to global variables: saves you a lot of time in loaded applications